### PR TITLE
[Docs] Update instructions for Microsoft SQL Server #6692

### PR DESCRIPTION
**Pull request by KelvinBrannonJr**
## Description

https://github.com/mindsdb/mindsdb/issues/6692#issue-1777011012

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: docs/data-integrations/sap-sql-anywhere.mdx [hunk 1]
```diff
@@ -21,9 +21,8 @@ The required arguments to establish a connection are as follows:
 * `server` sets the current server.
 
 <Tip>
-If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the handler's folder (mindsdb/integrations/handlers/sqlany_handler) and run this command: `pip install -r requirements.txt`.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/sap-sql-anywhere.mdx:24]: Please undo this change.

**Comment by martyna-mindsdb** [comment]
@KelvinBrannonJr 
As mentioned before, you need to update just the `docs/data-integrations/microsoft-sql-server.mdx` file.

Please undo the changes you made to the `docs/data-integrations/sap-sql-anywhere.mdx` file - see the review comment above.

You can push the changes to this PR instead of opening a new one. Please familiarize yourself with the GitHub flow: https://docs.github.com/en/get-started/quickstart/github-flow

File: docs/data-integrations/sap-sql-anywhere.mdx (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/data-integrations/sap-sql-anywhere.mdx:24]: Again, the changes to this file should be undone by using `git rm sap-sql-anywhere.mdx` and commiting/pushing to this PR.

Comment (KelvinBrannonJr) [reply docs/data-integrations/sap-sql-anywhere.mdx:24]: I'm confused did you want me to add the text back into sap-sql-anywhere? Originally you said to delete it??

**Comment by martyna-mindsdb** [comment]
@KelvinBrannonJr 
Again, the changes to the `sap-sql-anywhere.mdx` file should be undone by using `git rm sap-sql-anywhere.mdx` and commiting/pushing to this PR.

Here is how you can verify whether you completed it as requested:
- In the `Files changed` tab above, there are two files that you modified.
- But there should be only one file.

File: docs/data-integrations/sap-sql-anywhere.mdx (hunk 1 above)
Comment (KelvinBrannonJr) [reply docs/data-integrations/sap-sql-anywhere.mdx:24]: ok what is the git command to push directly to a specific PR?

**Comment by KelvinBrannonJr** [comment]
The second file is the staged the fix from issue 6692 from
before docs/data-integrations/microsoft-sql-server.mdx

File: docs/data-integrations/sap-sql-anywhere.mdx (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/data-integrations/sap-sql-anywhere.mdx:24]: I said to undo your changes - not delete the content of this file.

You can do as follows:
`git rm sap-sql-anywhere.mdx` - to delete the file
`git add *` - to add to the branch
`git commit -m "changes description"` - to commit to the branch
`git push` - to push to this PR

Please see the GitHub docs I linked earlier.
